websocket_server/websocket_server.py

```diff
@@ -84,9 +84,9 @@ def set_fn_message_received(self, fn):
     def send_message(self, client, msg):
         self._unicast_(client, msg)
 
-    def send_message_to_all(self, msg):
-        self._multicast_(msg)
-
+    @classmethod
```

Comment:
Not required to override this. Fine for `send_message_to_all` to just unicast to clients for the instance